docs/community/software/node-red-messages.mdx

```diff
@@ -0,0 +1,37 @@
+---
+id: community-node-red-messages
+title: Node-RED Messages Node
+sidebar_label: Node-RED Messages
+sidebar_position: 4
+---
+
+:::note
+
+This is a community project maintained by @danpeig.
+Development can be followed on [GitHub](https://github.com/danpeig/node-red-meshtastic-msg).
+Support should be sought from the respective authors.
+
+:::
+
+## Overview
+
+This [Node-RED](https://nodered.org) node allows sending and receiving packets to a Meshtastic mesh network thru a device connected via HTTP.
+It is based on [Meshtastic.js](https://js.meshtastic.org/) library.
+
+## Features
+- Send and receive text messages to/from any device in the mesh
+- Supervise/monitor device status
+- Subscribe and receive all types of events supported by [Meshtastic.js](https://js.meshtastic.org/) (examples: Atak, Position, Range Test, Map Report, Store and Forward, etc...)
+- Send packets to any Meshtastic APP (port num)
+- Plug and Play: no additional servers, no containers, no CLI, no binary files, super simple configuration
+- Indirect support to MQTT via uplink/downlink channels
+
+## Limitations
+- Connect directly to the MQTT server: this is not supported by [Meshtastic.js](https://js.meshtastic.org/)
+- Connect to a device via Bluetooth or Serial (not implemented yet)
+
+## Instructions
+For installation and usage instructions, please visit the project page at [GitHut](https://github.com/danpeig/node-red-meshtastic-msg).
+
+## Screenshot
+![Node-RED Meshtastic Messages node](https://github.com/danpeig/node-red-meshtastic-msg/raw/main/resources/flow_example.png)
```

Comment:
I believe it's preferred to place images in
https://github.com/meshtastic/meshtastic/tree/master/static/img

Also webp format is preferred\.

Reply:
Fixed!